README.md

```diff
@@ -18,14 +18,14 @@ docker run --detach --name py_notes_project \
   mongo:6.0.6-jammy
 ```
 ### Requirements
-* `Python` 3.9+ (tested to work with >= 3.11.2)  
-* `Django` (tested to work with >= 4.2.1)  
-* `Flask` (tested to work with >= 2.3.2)  
-* `django-bootstrap-v5` (tested to work with >= 1.0.11)   
-* `mongomock` (tested to work with >= 4.1.2)  
-* `pymongo` (tested to work with >= 4.3.3)  
-* `pytz` (tested to work with >= 2023.3)  
-* `requests` (tested to work with >= 2.31.0)  
+* `Python` 3.9+ (tested to work with == 3.11.2)  
+* `Django` (tested to work with == 4.2.1)  
+* `Flask` (tested to work with == 2.3.2)  
```

Comment:
Usually, it's either Django or Flask, not both. Any specific reason for using them both in this project?

Reply:
Django is used in "web view" part. PyNotes runs on Django server.
Flask is used in "REST API" part. Flask handles HTTP requests of Django's views.py.

I started learning GraphQL, so graphene (graphene_mongo) packages will be used instead of Flask.